src/components/tabGroup/tabGroup.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { BorderDirection } from '../../core/types/directions';
+import { SizeType } from '../../core/types/size';
+import IonTab, { TabProps } from '../tab/tab';
+import { TabGroupStyles } from './styles';
+
+type TabAlign = 'vertical' | 'horizontal';
+
+export type TabGroupSizes = Exclude<SizeType, 'xs'>;
+
+export interface IonTabGroupProps {
+  tabs: TabProps[];
+  selected?: boolean;
+  align?: TabAlign;
+  size?: TabGroupSizes;
+  borderDirection?: BorderDirection;
+}
+
+const IonTabGroup = ({
+  tabs,
+  size = 'sm',
+  align = 'horizontal',
+  borderDirection = 'bottom',
+}: IonTabGroupProps) => {
+  const [activeTab, setActiveTab] = useState<number>();
+
+  const handleTabClick = useCallback((index: number) => {
+    setActiveTab(index);
+  }, []);
+
+  const defineBorderByAlign = () => {
+    const defaultHorizontal = 'bottom';
+    const defaultVertical = 'right';
+
+    if (['top', 'bottom'].includes(borderDirection)) {
```

Comment:
some edge position settings are different than Ion in Angular. It would be interesting to check the possibilities.

eg: the Ion angular I can have a horizontal tabgroup with a right or left border. what is not possible to do here.